subsys/http/src/main/java/org/commonjava/indy/subsys/http/IndyHttpProvider.java

```diff
@@ -83,55 +84,51 @@ public Http getHttp()
         return http;
     }
 
-    public HttpClientContext createContext()
-            throws IndyHttpException
+    @Deprecated
```

Comment:
I don't think these are exposed anywhere in a public API, and since we don't have a true plugin model (add-ons have to be bundled into the application), I don't think it's dangerous to simply take out these methods we're no longer interested in, and replace them with the new calls.

We will have to deprecate / provide backward compat for anything in the REST API or Java client API...otherwise, I don't think it helps us much to maintain it.